src/events/default-event-dispatcher.ts

```diff
@@ -135,7 +136,10 @@ export function newDefaultEventDispatcher(
   const sdkKeyDecoder = new SdkKeyDecoder();
   const ingestionUrl = sdkKeyDecoder.decodeEventIngestionHostName(sdkKey);
   if (!ingestionUrl) {
-    throw new Error('Unable to parse Event ingestion URL from SDK key');
+    logger.info(
```

Comment:
Consider `logger.debug` to keep logs to a minimum since this would be visible in the browser console